.github/workflows/ci-tests.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     name: CCM Integration tests
     uses: ./.github/workflows/integration-tests.yml
     with:
-      reloc_version: "2023-04-13T13:31:00Z"
+      reloc_version: "2025-01-19T09:39:05Z"
```

Comment:
it won't be enough, one would need to update the tests as well

also I think maybe pinning down the runner ubuntu version would help for now

Reply:
I managed to fix it by using java11 and newer version of the build. I however had to change the version in the tests as well and fixed some test bugs I encountered as well. I will extract the changes into a separate PR